selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/forum/ForumLocator.java

```diff
@@ -891,4 +891,11 @@
   public final By                     ELEMENT_MOVE_TOPIC_POPUP                           = By.id("UIForumPopupWindow");
 
   public static final SelenideElement ELEMENT_BUTTON_SUBMIT_POST= $(byXpath("//*[@id=\"UIPostForm\"]/div[3]/button[1]"));
+
+
+  public static final SelenideElement ELEMENT_TOPIC_MORE_ACTION= $(byXpath("//*[@id=\"UITopicDetail\"]/div[4]/div[1]/ul/li[3]/div/div[1]"));
```

Comment:
1.locator already exist (ELEMENT_MORE_ACTION) 
2. This locator should be deleted after using the function (forumTopicManagement.selectItemMoreActionMenuTopic(specifMoreActionMenuTopic.LOCK);)